### PR TITLE
Add install instructions and remove both plugins from Backstage

### PR DESCRIPTION
**Pull request by mareklibra**
This patch removes the `notifications-frontend` and `notifications-backend` plugins from the Backstage instance and adds instructions how to add them into a target one.

For the further development purposes, there is a reverse commit to be applied on a feature branch but can not be merged here: https://github.com/mareklibra/janus-idp-backstage-plugins/commits/register.plugins

---
FLPATH-708

### CODE REVIEW

File: plugins/notifications-backend/README.md [hunk 1]
```diff
@@ -1,9 +1,13 @@
 # Notifications
 
-Welcome to the Notifications backend plugin!
+This Backstage back-end plugin provides REST API endpoint for the notifications.
```
Comment (ydayagi) [inline plugins/notifications-backend/README.md]: backend

File: yarn.lock
Comment (ydayagi) [inline yarn.lock:1]: i think you need to remove this file

Comment (mareklibra) [reply yarn.lock:1]: I hoped we will not need any `yarn.lock` changes till the very last moment. Unfortunately, there is conflict around the knex 2.4.2 and 2.5.1. 

The notifications-backend require `knex 2.5.1`  due to the `uuids`
The rbac plugin relies on `2.4.2`, the `tsc` fails without unifying.

Trying to put this change to a separate PR now.

Comment (mareklibra) [reply yarn.lock:1]: Done: https://github.com/mareklibra/janus-idp-backstage-plugins/pull/44

**Review by ydayagi — APPROVED** [review]

**Pull request merged**